dandi/consts.py

```diff
@@ -132,6 +135,18 @@ def urls(self) -> Iterator[str]:
         f"http://{instancehost}:8085",
         f"http://{instancehost}:8000/api",
     ),
+    "linc": DandiInstance(
+        "linc",
+        "https://lincbrain.org",
+        "https://api.lincbrain.org/api",
+        is_private=True
+    ),
+    "linc-staging": DandiInstance(
+        "linc-staging",
+        "https://staging.lincbrain.org",
```

Comment:
Hi @aaronkanzer, it looks like https://staging.lincbrain.org isn't able to resolve again.  Not sure if we changed anything.  See screenshot.

<img width="689" alt="image" src="https://github.com/user-attachments/assets/03dac921-9bb6-47cb-8631-d6a42ec3d2bc">

Reply:
![Screenshot 2024-11-12 at 12 24 18 PM](https://github.com/user-attachments/assets/aaca7a8c-7eab-4aaf-9034-5e5188c2f031)

@kabilar -- it seems that SSL support isn't provided out-of-the-box for the branch deploys URL

Firefox and Chrome require the SSL enforcement, but if you use Safari, you can proceed without the SSL -- didn't want to disturb the peace here with changing the SSL cert, so didn't update anything for now